spid_cie_oidc/relying_party/templates/rp_landing.html

```diff
@@ -63,6 +63,18 @@ <h3 class="text-left">
                                     {% endif %}
                                 </a>
                             </li>
+                         <li>
+                            <a href="{% url 'spid_cie_rp_begin' %}?provider={{sub}}&profile=spid&scope=openid offline_access" id="provider-spid-refresh_token">
+                                <span class="spid-sr-only">{{ attrs.organization_name }} SESSIONE A LUNGO TERMINE</span>
```

Comment:
as well here, `{% trans "" %}` block is then required for every text string given to the users